README.md

```diff
@@ -7,33 +7,34 @@
 **ДИСКЛЕЙМЕР**: целью было быстро сделать прототип. Писалось всё быстро и на коленке, код дурно пахнет и на 100% подлежит переписыванию.
 
 ## Разработка
+
 ### Настройка
+
 Файл `.env.sample` необходимо скопировать в `.env` и заполнить или скорректировать необходимые значения.
 
 В hosts добавить:
-```
+
+```hosts
 127.0.0.1 orbitar.local api.orbitar.local
 ```
-Можно дополнить списком подсайтов по вкусу (опционально)
-```
-127.0.0.1 idiod.orbitar.local
-```
 
 ### Особенности
 
 * [Локальный запуск в дебаг режиме](docs/local-development.md)
 * [Организация .env переменных](docs/environment.md)
 
-
 ### Запуск production-сборки полностью в контейнере
 
 Пересборка фронта и бэка (если необходимо):
+
+```sh
+docker compose -p orbitar -f docker-compose.yml -f docker-compose.deploy.yml -f docker-compose.deploy.nonssl.yml build --no-cache frontend backend
 ```
-docker compose -p orbitar -f docker-compose.local.yml build --no-cache frontend backend
-```
+
 Запуск:
-```
-docker compose -p orbitar -f docker-compose.local.yml up
+
+```sh
+docker compose -p orbitar -f docker-compose.yml -f docker-compose.deploy.yml -f docker-compose.deploy.nonssl.yml up
```

Comment:
can we use `include` to avoid specifying multiple files here?

https://docs.docker.com/compose/how-tos/multiple-compose-files/include/

Reply:
I'll take a look. IIRC, includes can't conflict.

Reply:
Renamed base files and made real ones consist of includes